src/core/localization/translations/en/common-messages.json

```diff
@@ -1,19 +1,180 @@
 {
   "common": {
-    "Request Timeout": "Request Timeout!",
-    "No administrative boundaries": "No administrative boundaries here!",
-    "Self intersections are not supported": "Self intersections are not supported!",
+    "km": "km",
+    "m": "m",
+    "to": "to",
 
-    "bivariate.matrix.icon.population": "Value divided by Population",
-    "bivariate.matrix.icon.area_km2": "Value divided by Area",
-    "bivariate.matrix.icon.total_building_count": "Value divided by Total Buildings Estimate",
-    "bivariate.matrix.icon.populated_area_km2": "Value divided by Populated Area",
-    "bivariate.matrix.icon.one": "Without divider",
-    "bivariate.matrix.icon.roads": "Value divided by Total Roads Length",
+    "save": "Save",
+    "cancel": "Cancel",
+    "create": "Create",
 
-    "bivariate.legend.high": "High",
-    "bivariate.legend.low": "Low",
+    "disasters": "Disasters",
+    "loading": "Loading...",
+    "loading_events": "Loading Events",
+    "about": "About",
+    "to_main_page": "to main page",
+    "legend": "Legend",
+    "maps": "maps",
+    "vertical_direction": "Vertical Direction",
```

Comment:
There's theoretical concern about leading capital letters. I think it wouldn't be actual problem as we don't have so much text elements, but can you confirm it's okay?

Reply:
I think it's okay, but anyway, I noticed it is not capital on designs so I turned it back noncapital.

Reply:
nice catch @IlyaIzr ! I've rechecked the strings one more time.